CMakeModules/FGInstallDirs.cmake

```diff
@@ -40,7 +40,7 @@ endif()
 
 # CMake files
 if(NOT DEFINED FG_INSTALL_CMAKE_DIR)
-    set(FG_INSTALL_CMAKE_DIR "cmake" CACHE PATH "Installation path for CMake files")
+    set(FG_INSTALL_CMAKE_DIR "lib/cmake/Forge" CACHE PATH "Installation path for CMake files")
```

Comment:
Just `cmake` works for Windows OS. I was testing this on Windows initially, thus forgot set it for *nix platforms. Can you please add a check for WIN32 and set it to `cmake` and leave `lib/cmake/Forge` for the rest.

Reply:
Sure.